packages/zoltan2/test/correctness/zoltanCompare.cpp

```diff
@@ -135,14 +145,18 @@ static void zobjlist(void *data, int ngid, int nlid,
 {
   *ierr = ZOLTAN_OK;
   tMVector_t *vec = (tMVector_t *) data;
-  int n = vec->getLocalLength();
-  for (int i = 0; i < n; i++) {
-    gids[i] = vec->getMap()->getGlobalElement(i);
+  size_t n = vec->getLocalLength();
+
+  for (size_t i = 0; i < n; i++) {
+    zgno_t vgid = vec->getMap()->getGlobalElement(i);
```

Comment:
This may issue a warning, since it casts `i` from unsigned `size_t` to signed `LO`.